### PR TITLE
Liquibase migration

### PR DESCRIPTION
**Pull request by VovaProtsiv**
Added Liquibase for managing database schema changes.

### CODE REVIEW

File: src/main/resources/db/db.changelog-setup.xml [hunk 1]
```diff
@@ -0,0 +1,267 @@
...
+        <addPrimaryKey
+                columnNames="feedback_request_id, question_id"
+                constraintName="course_feedback_request_questions_pkey"
+                schemaName="public"
+                tableName="course_feedback_request_question"
+        />
+    </changeSet>
+
+    <changeSet id="1.0.15" author="volodymyr.protsiv">
+        <addForeignKeyConstraint baseColumnNames="role_id" baseTableName="user_roles" constraintName="user_roles_roles_fkey" deferrable="false" initiallyDeferred="false" onDelete="NO ACTION" onUpdate="NO ACTION" referencedColumnNames="id" referencedTableName="role"/>
```
Comment (Forfend) [inline src/main/resources/db/db.changelog-setup.xml:221]: Please, add these constraints directly in changeset where you create tables

**Review by Forfend — CHANGES_REQUESTED** [review]

File: src/main/resources/db/db.changelog-setup.xml (hunk 1 above)
Comment (VovaProtsiv) [reply src/main/resources/db/db.changelog-setup.xml]: Added constraints directly. 
Added db.changelog-insert-role-permission.xml with role and permission insertions. @Forfend

File: src/main/resources/db/changelog/db.changelog-insert-role-permission.xml [hunk 2]
```diff
@@ -0,0 +1,45 @@
+<databaseChangeLog xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+                   xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+                   xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog
+                        http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.4.xsd">
+
+    <changeSet id="1.1.0" author="volodymyr.protsiv">
```
Comment (Forfend) [inline src/main/resources/db/changelog/db.changelog-insert-role-permission.xml:6]: Can we have ```sql``` file with these data for insert?

**Review by Forfend — CHANGES_REQUESTED** [review]

File: src/main/resources/db/changelog/db.changelog-insert-role-permission.xml (hunk 2 above)
Comment (VovaProtsiv) [reply src/main/resources/db/changelog/db.changelog-insert-role-permission.xml:6]: Added role_permission.sql. 
In db.changelog-insert-role-permission.xml I used an sqlFile tag to execute insert statments.

**Review by Forfend — APPROVED** [review]

**Pull request merged**